src/pointfree/bichain.js

```diff
@@ -0,0 +1,21 @@
+/** @license ISC License (c) copyright 2019 original and current authors */
+/** @author Dale Francis (dalefrancis88) */
+
+const curry = require('../core/curry')
+const isBichain = require('../core/isBichain')
+const isFunction = require('../core/isFunction')
+
+// bichain : bichain m => (e -> m c b) -> (a -> m c b) -> m e a -> m c b
+function bichain(f, g, m) {
+  if(!isFunction(f) || !isFunction(g)) {
+    throw new TypeError('bichain: First two arguments must be Async returning functions')
```

Comment:
This should read:
`First two arguments must be Bicahin returning functions of the same type`

Reply:
@evilsoft is this because we're going to put this on the others?

Reply:
Yep!